mcs/build/profiles/mobile.make

```diff
@@ -16,7 +16,6 @@ DEFAULT_REFERENCES = -r:$(topdir)/class/lib/$(PROFILE)/mscorlib.dll
 PROFILE_MCS_FLAGS = \
 	-d:NET_1_1 \
 	-d:NET_2_0 \
-	-d:NET_2_1 \
```

Comment:
I am not sure, we can keep it now but I'd still like to remove it later as it'll become confusing what is used and what is dead